client/Components/GameBoard/PlayerRow.jsx

```diff
@@ -154,33 +155,43 @@ class PlayerRow extends React.Component {
                     size={ this.props.cardSize } />
                 { this.getAgenda() }
```

Comment:
Need to make Conclave be draggable as well (but probably not other agenda types). I imagine that it's still setting the now unused `onDragDrop` property in this method since I don't see a diff.